content/en/docs/reference/controllers/resource-quota.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Resource quota controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+This controller limits the quantity of objects that can be created in a namespace
+by object type, as well as the total amount of compute resources that may be
+consumed by resources in that project.
```

Comment:
"that project" is used, but there is no mention of "project" earlier in the paragraph. This is likely due to legacy Google-specific terminology? Probably best to rephrase it here.